core/src/main/java/org/mskcc/oncotree/utils/SlackUtil.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2019 Memorial Sloan-Kettering Cancer Center.
+ *
+ * This library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY, WITHOUT EVEN THE IMPLIED WARRANTY OF
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE.  The software and
+ * documentation provided hereunder is on an "as is" basis, and
+ * Memorial Sloan-Kettering Cancer Center
+ * has no obligations to provide maintenance, support,
+ * updates, enhancements or modifications.  In no event shall
+ * Memorial Sloan-Kettering Cancer Center
+ * be liable to any party for direct, indirect, special,
+ * incidental or consequential damages, including lost profits, arising
+ * out of the use of this software and its documentation, even if
+ * Memorial Sloan-Kettering Cancer Center
+ * has been advised of the possibility of such damage.
+*/
+package org.mskcc.oncotree.utils;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClientBuilder;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SlackUtil {
+    private static final Logger logger = LoggerFactory.getLogger(SlackUtil.class);
+
+    @Value("${slack.url}")
+    private String slackURL;
+
+    @Value("${slack.channel}")
+    private String slackChannel;
```

Comment:
@averyniceday the slack channel needs a "#" in front of it in the payload below. It might be a good idea to keep it hardcoded below and just have the plain channel name in the properties file. (i.e., `slack.channel=msk-pipeline-logs` instead of `slack.channel=#msk-pipeline-logs`)